homeassistant/components/leviosa_shades/services.yaml

```diff
@@ -0,0 +1,13 @@
+next_down_pos:
```

Comment:
Best to remove this from your initial PR. It is not required as base functionality and can be added in a follow-up PR.

In the future, it would be good to add comments here to clarify what this does. See https://developers.home-assistant.io/docs/dev_101_services/#service-descriptions.

Reply:
There is ambiguity here, what it really means is for the motor of the cover to advance the cover to the next `position`, I will update the naming of this service to make it easier to understand and avoid ambiguity.

Reply:
Best to add this to a follow-up PR to keep the initial PR as small as possible.

Reply:
Are you referring to removing service.yaml from the current PR?

Reply:
Best is to remove all custom services. The smaller you keep the initial PR, the faster it can be merged to core. For your initial PR keep only the bare minimum required to connect to your device and one single entity. In a follow-up PR you can add additional sensors and services.

Reply:
ok, I just removed all services